ui/shared/components/panel/variants/Variants.jsx

```diff
@@ -68,14 +86,76 @@ const Variant = ({ variant }) => {
   )
 }
 
-Variant.propTypes = {
+SingleVariant.propTypes = {
   variant: PropTypes.object,
 }
 
+const CompoundHet = ({ variant }) => {
```

Comment:
this is almost a copy-paste of `SingleVariant` so you should modify that component to work for both (maybe add a `showGeneInfo` prop that is true for regular variants and false for compound hets or something like that